MDANSE_GUI/Src/MDANSE_GUI/Tabs/Views/PlotDataView.py

```diff
@@ -174,12 +157,28 @@ def quick_plot_data(
             Data model items collected for plotting.
         mda_data_structure : MDADataStructure
             The common HDF5 file from which the datasets originate.
+        main_only: bool
+            if True, only plot datasets with the 'main' tag
 
         """
         model = PlottingContext()
         file = mda_data_structure._file
         for data_node in data_nodes:
-            dataset = SingleDataset(data_node.child_path, file)
+            if not data_node.child_path:
+                continue
+            if main_only:
+                try:
+                    tags = file[data_node.child_path].attrs["tags"]
+                except KeyError:
+                    continue
+                if "main" not in tags:
+                    continue
+                if "partial" in tags:
+                    dataset = SingleDataset(data_node.child_path, file, linestyle="--")
+                else:
+                    dataset = SingleDataset(data_node.child_path, file)
+            else:
+                dataset = SingleDataset(data_node.child_path, file)
```

Comment:
```suggestion
                if "partial" in tags:
                    dataset_kwargs.update({"linestyle": "--"})
            model.add_dataset(
                SingleDataset(data_node.child_path, file, **dataset_kwargs)
            )
```

If we store a dictionary dataset_kwargs = {} at the top, we could save a couple of lines of if/else statements here.

Reply:
Doesn't even need to be kwargs, because we know the default (`-`), can just set style:
```python
style = "--" if "partial" in tags else "-"
dataset = SingleDataset(data_node.child_path, file, linestyle=style)
```